Makefile

```diff
@@ -47,7 +47,9 @@ endif
 ifneq ($(strip $(GLOBAL_CONSOLE_DISPLAY_NAME)),)
         MAVEN_ARGS+="-Dapplication.globalconsole.display.name=$(GLOBAL_CONSOLE_DISPLAY_NAME)"
 endif
-
+ifneq ($(strip $(PROJECT_DOCS_URL)),)
```

Comment:
I don't think we want to reply on arguments to the build to customise labels.  Better to include these in pom.properties.   That way derivative versions can specify their own values and subject them to source control.